### PR TITLE
check before copy

### PR DESCRIPTION
**Pull request by lemire**

### CODE REVIEW

**Comment by jkeiser** [comment]
Ok we really really need to just use cmake for this instead of going back and forth. It seems like exactly what a make system is for ...

**Comment by lemire** [comment]
FreeBSD support remains broken. I will try reverting the bash script change.

File: singleheader/amalgamate.sh [hunk 1]
```diff
@@ -127,8 +127,17 @@ echo "/* auto-generated on ${timestamp}. Do not edit! */" > ${AMAL_C}
     done
 } >> ${AMAL_C}
 
-cp -f "${SCRIPTPATH}/amalgamate_demo.cpp" "${DEMOCPP}"
-cp -f "${SCRIPTPATH}/README.md" "${README}"
+function docopy()
+{
+  if [ "$1" -ef "$2" ]; then
+    echo "I would copy $1 to $2 but they are the same file.";
```
Comment (pps83) [inline singleheader/amalgamate.sh:133]: there is no need to log that message. All we really want to do is to copy these files if output dir isn't the same as singleheader dir. See udpated PR: https://github.com/simdjson/simdjson/pull/1113/files

**Comment by lemire** [comment]
Closing. We went with another PR by @pps83  instead.

**Pull request closed without merging**